_data/team.yml

```diff
@@ -199,3 +199,13 @@
     <p>Alvaro has been working in the IT industry for product companies, consulting firms and as a freelancer since 2005.</p>
   twitter: alvarobiz
   github: alvarogarcia7
+
+- id: carlos
+  name: Carlos Morera de la Chica
+  image: /assets/img/custom/team/carlos_morera_de_la_chica.jpg
+  role: Apprentice
+  shortDescription: Carlos is a software developer who is constantly looking for new ways to master the art of crafting software. He firmly believes in the software craftsmanship principles and bases his journey towards mastery on them.
+  fullDescription: <p>Carlos is a software developer who is constantly looking for new ways to master the art of crafting software. He firmly believes in the software craftsmanship principles and bases his journey towards mastery on them.</p>
```

Comment:
you have software in the first sentence twice. Maybe remove it from `software developer`?